pkg/receive/handler.go

```diff
@@ -1,18 +1,25 @@
 package receive
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"io/ioutil"
 	"net/http"
 	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/snappy"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/prometheus/prompb"
 
 	"github.com/openshift/telemeter/pkg/authorize"
 )
 
 const forwardTimeout = 5 * time.Second
+const RequestLimit = 15 * 1024 // based on historic Prometheus data with 6KB at most
```

Comment:
can we make this `DefaultRequestLimit`? Eventually we'll make the request limit overridable via flags, no?

Reply:
Yes. :blush: